### DISCUSSION TITLE
Upcoming roadmap

### DISCUSSION THREAD

**Discussion by mrpea** [Netflix/dispatch#3117] [General]
Hi - do you have any tentative roadmap that you can share? 

Also how open are you to contributions adding more visual interactions to the slackbot?

**Comment by mvilanova** [comment]
@mrpea in the next coming months we will continue to invest in improving dashboard metrics for cases and incidents, signal processing, adding support for incident workstreams, and team health metrics. Feel free to open PRs for the additional interactions for the Slack plugin and we can discuss there.
